client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -11,14 +11,38 @@
  * External dependencies
  */
 import { Component } from '@wordpress/element';
-import { Disabled, FormToggle } from '@wordpress/components';
+import { Disabled, FormToggle, Notice } from '@wordpress/components';
 import { withSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import PublicizeServiceIcon from './service-icon';
 
+/**
+ * If this is the Google+ connection, display a notice.
+ * ToDo: change 'twitter' to 'google-plus' before merging.
+ *
+ * @param {string} serviceName Name of the connnected social network.
+ * @returns {object} Message warning users of Google+ shutting down.
+ */
+const maybeDisplayGooglePlusNotice = serviceName =>
+	'twitter' === serviceName && (
+		<Notice status="error">
+			{ __(
+				'Google+ will shut down in April 2019. You can keep posting with your existing Google+ connection through March.'
+			) }
+			<a
```

Comment:
What do you think about using Gutenberg ExternalLink component here? Would be consistent by giving us an icon, too.

Reply:
Didn't know about it. Trying it now. Thanks for pointing it.